venues/aclweb.org/ACL/migrate_submissions_ARR_ACL.py

```diff
@@ -0,0 +1,776 @@
+import argparse
+from re import sub
+import openreview
+from tqdm import tqdm
+import csv
+
+"""
+OPTIONAL SCRIPT ARGUMENTS
+
+    baseurl -  the URL of the OpenReview server to connect to (live site: https://openreview.net)
+    username - the email address of the logging in user
+    password - the user's password
+
+"""
+parser = argparse.ArgumentParser()
+parser.add_argument('--baseurl', help="base URL")
+parser.add_argument('--username')
+parser.add_argument('--password')
+args = parser.parse_args()
+client = openreview.Client(baseurl=args.baseurl, username=args.username, password=args.password)
+
+
+# Posting submission invitation 
+submission_invitation = openreview.Invitation(
+    id = "aclweb.org/ACL/2022/Conference/-/Submission",
+    signatures = ["aclweb.org/ACL/2022/Conference"],
+    readers = ["everyone"],
+    invitees=["aclweb.org/ACL/2022/Conference"],
+    reply={
+        "readers":{
+            "values":[
+                "aclweb.org/ACL/2022/Conference"
+            ]
+            },
+        "writers":{
+            "values":[
+                "aclweb.org/ACL/2022/Conference"
+                ]
+            },
+        "signatures":{
+            "values":[
+                "aclweb.org/ACL/2022/Conference"
+                ]
+            },
+        "content":{
+            "title": {
+                "description": "Title of paper. Add TeX formulas using the following formats: $In-line Formula$ or $$Block Formula$$",
+                "order": 10,
+                "value-regex": ".{1,250}",
+                "required": True
+            },
+            "authors": {
+                "description": "Comma separated list of author names.",
+                "order": 11,
+                "values-regex": "[^;,\\n]+(,[^,\\n]+)*",
+                "required": True,
+                "hidden": True
+            },
+            "authorids": {
+                "description": "Search author profile by first, middle and last name or email address. If the profile is not found, you can add the author by completing first, middle, and last names as well as author email address.",
+                "order": 12,
+                "values-regex": "~.*|([a-z0-9_\\-\\.]{1,}@[a-z0-9_\\-\\.]{2,}\\.[a-z]{2,},){0,}([a-z0-9_\\-\\.]{1,}@[a-z0-9_\\-\\.]{2,}\\.[a-z]{2,})",
+                "required": True
+            },
+            "abstract": {
```

Comment:
this is mostly cosmetic (and not a big deal), but I think abstract should go right after the title and authorids, then the rest of the fields (and maybe we can leave the comment to SACs last?) just seems like a most natural order of the fields